ee/tabby-webserver/src/schema/analytic.rs

```diff
@@ -0,0 +1,53 @@
+use std::fmt::Display;
+
+use async_trait::async_trait;
+use chrono::{DateTime, Utc};
+use juniper::{GraphQLEnum, GraphQLObject, ID};
+
+use crate::schema::Result;
+
+#[derive(GraphQLObject, Debug)]
+pub struct CompletionStats {
+    pub start: DateTime<Utc>,
+    pub end: DateTime<Utc>,
+
+    pub completions: i32,
+    pub selects: i32,
+}
+
+// FIXME(boxbeam): Adding more languages.
```

Comment:
Please create task for this fixme indicating which languages and what needs to be added

Reply:
https://github.com/TabbyML/tabby/blob/1e0d0e3014c365ada70a14b4c30639ee43bd65ce/crates/tabby-scheduler/src/dataset.rs#L158 is a good list to start with.